src/crowd.go

```diff
@@ -1,37 +1,47 @@
 package main
 
 import (
+	"errors"
 	"flag"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"net/http"
 
-	"github.com/aws/aws-sdk-go/service/sqs/sqsiface"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/14-bits/crowd/pkg/sink"
 	"github.com/julienschmidt/httprouter"
 )
 
 var port int
 var endpoint string
-var queue string
+var sinkType string
+var sinkUrl string
 
 func init() {
 	flag.IntVar(&port, "port", 2104, "HTTP Server Port")
 	flag.StringVar(&endpoint, "endpoint", "/api/foo", "HTTP path to receive POST requests")
-	flag.StringVar(&queue, "queue", "https://sqs.eu-central-1.amazonaws.com/21042018/foo", "URL for the SQS queue")
+	flag.StringVar(&sinkType, "sink-type", "void", "Sink adapter, e.g.: void, sqs, redis...")
```

Comment:
I think it’s a nice idea, but if you have basic with it would work... we could structure it differently perhaps: `redis://...`, but it could also be a problem to parse in the case of protocol specific sinks ...

Reply:
Let’s open an issue to think it through